pkg/hubble/relay/pool/manager.go

```diff
@@ -236,20 +236,40 @@ func (m *PeerManager) List() []poolTypes.Peer {
 	return peers
 }
 
-func (m *PeerManager) add(hp *peerTypes.Peer) {
+func (m *PeerManager) put(hp *peerTypes.Peer) {
```

Comment:
nit, informal (i.e. non-blocking): `put` is absolutely fine, but I wanted to point out that most other code in the Cilium codebase uses the name `upsert` for such functions

Reply:
I personally find `upsert` a bit fuggly so I usually use `put`, but if `upsert` is more common let's rename it to `upsert`.